programs/openbook_v2/tests/integration.rs

```diff
@@ -0,0 +1,319 @@
+                                client_order_id: 0,
+                                order_type: PlaceOrderType::Limit,
+                                expiry_timestamp: 0,
+                                self_trade_behavior: SelfTradeBehavior::DecrementTake,
+                                limit: 10,
+                            },
+                            &market_keys,
+                            &ooa,
+                            &base_mint,
+    ).await?;
+    // init drift
+    let (state, drift_signer) = initialize_drift(&mut banks_client, &keypair, &quote_mint).await?;
+    println!("state: {} drift signer: {}", state, drift_signer);
+    // init quote market
+    let (quote_market, quote_market_vault, quote_insurance_fund_vault ) = init_quote_market(
+        &mut banks_client,
+        &keypair,
+        &quote_mint,
+        &drift_signer,
+        &state,
+    ).await?;
+
+    // pyth feed
+    let oracle_feed = initialize_pyth_oracle(&mut banks_client, &keypair,  PythInitialize{
+        price: 155_000,
+        expo: -3,
+        conf: 28988326,
+    }).await?;
+    println!("{}", oracle_feed);
+
+    // create spot market
+    let (spot_market, spot_market_vault, spot_insurance_fund_vault ) = init_spot_market(
+        &mut banks_client,
+        &keypair,
+        &base_mint,
+        &drift_signer,
+        &state,
+        &oracle_feed,
+    ).await?;
+
+    // create user
+    let (user, user_stats) = create_user(&mut banks_client, &keypair, &state).await?;
+
+    // deposit mock USDC  ...
+    deposit_and_execute(&mut banks_client,
+                        &keypair,
+                        Deposit{
+                            market_index: 0,
+                            amount: 1_000_000_000, // 1000 mock usdc
+                            reduce_only: false,
+                        },
+                        &state,
+                        &user,
+                        &user_stats,
+                        &quote_mint,
+                        &mut vec![AccountMeta::new(quote_market, false)]
+    ).await?;
+
+    deposit_and_execute(&mut banks_client,
+                        &keypair,
+                        Deposit{
+                            market_index: 1,
+                            amount: 1_000_000_000_000, // 1000 mock usdc
+                            reduce_only: false,
+                        },
+                        &state,
+                        &user,
+                        &user_stats,
+                        &base_mint,
+                        &mut vec![
+                            AccountMeta::new_readonly(oracle_feed, false),
+                             AccountMeta::new(spot_market, false),
+                             AccountMeta::new(quote_market, false),
+                        ]
+    ).await?;
+
+    // create OpenbookV2 fulfillment config
+    let config = initialize_openbook_v2_config(&mut banks_client,
```

Comment:
this fails for me with `AnchorError occurred. Error Code: InstructionFallbackNotFound. Error Number: 101. Error Message: Fallback functions are not supported.`, error code in hex is 0x65

Reply:
need more info from you to be able to resolve it, on my end works.